.circleci/config.yml

```diff
@@ -164,6 +167,20 @@ jobs:
           command: yarn test:ci
 
 
+  test_karma_mixed_shadow_mode:
```

Comment:
This job is standalone instead of being part of the `test_karma` job because we want this to be an optional check until we finish addressing all the test failures.